<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Капча</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QLineEdit" name="antigate_le"/>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="antigate_rb">
         <property name="text">
          <string>Antigate</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="captchabot_le">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="captchabot_rb">
         <property name="text">
          <string>CaptchaBot</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="ripcaptcha_rb">
         <property name="text">
          <string>RipCaptcha</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="ripcaptcha_le">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Настройки сервисов распознавания капчи для поисковых систем</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>antigate_rb</sender>
   <signal>toggled(bool)</signal>
   <receiver>antigate_le</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>antigate_rb</sender>
   <signal>toggled(bool)</signal>
   <receiver>captchabot_le</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>antigate_rb</sender>
   <signal>toggled(bool)</signal>
   <receiver>ripcaptcha_le</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>captchabot_rb</sender>
   <signal>toggled(bool)</signal>
   <receiver>antigate_le</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>captchabot_rb</sender>
   <signal>toggled(bool)</signal>
   <receiver>captchabot_le</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>captchabot_rb</sender>
   <signal>toggled(bool)</signal>
   <receiver>ripcaptcha_le</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>219</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ripcaptcha_rb</sender>
   <signal>toggled(bool)</signal>
   <receiver>antigate_le</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ripcaptcha_rb</sender>
   <signal>toggled(bool)</signal>
   <receiver>captchabot_le</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ripcaptcha_rb</sender>
   <signal>toggled(bool)</signal>
   <receiver>ripcaptcha_le</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
